src/core/server/http/http_service.ts

```diff
@@ -198,15 +198,13 @@ export class HttpService
       path: '/{p*}',
       method: '*',
       handler: (req, responseToolkit) => {
-        this.log.debug(
-          `OpenSearch Dashboards server is not ready yet ${req.method}:${req.url.href}.`
-        );
+        this.log.debug(`Wazuh Dashboard server is not ready yet ${req.method}:${req.url.href}.`);
```

Comment:
**nitpick**: `Dashboard` should be in lowercase.

Reply:
To be discussed.